src/commands/release/constants.js

```diff
@@ -16,7 +16,14 @@ module.exports.MESSAGES = {
   FORCE_REMINDER,
   HAS_CHANGES: `You shouldn't release builds which may contain un-committed changes! ${FORCE_REMINDER}`,
   NOT_REPO: `You can't tag a release or deploy using a tag name becase this project isn't a git repo.`,
-  createBump: (bump, from, to) => `Bump ${hvy(bump)} version ${hvy(from)}=>${hvy(to)}`,
+  changes: (tag, tags, changelog) =>
+    changelog.length
+      ? `${tags.length ? `Here's what's changed since ${hvy(tag)}` : 'Here are the initial changes'}:
+
+${changelog.join('\n')}
+`
+      : `${cmd('ℹ')} Nothing has changed since ${hvy(tag)}\n`,
+  createCommit: (from, to) => `Bump version ${hvy(from)}=>${hvy(to)}`,
```

Comment:
Can we add some space around the `=>`?

eg. `Bump version 1.0.0-pre => 1.0.0`

Reply:
[uh huh](https://github.com/abcnews/aunty/pull/71/commits/62c395d2294dd3a435531b3e86d1e2b5c3818d32)